app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import NotVaccinatedError,\
+    NotWearingMaskError,\
+    OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+        self.today = datetime.date.today()
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" in visitor.keys():
```

Comment:
You can simplify this function and write without nested ifs.
To do this, check first if the "vaccine" key is **not** present in the dict.
```
if "vaccine" not in visitor.keys():
     raise NotVaccinatedError("Some message")
```